src/prommis/uky/uky_flowsheet.py

```diff
@@ -1528,31 +2960,23 @@ def add_costing(flowsheet):
         },
     )
 
-    # 3.2 is UKy Roasting - Conveyors
-    R_conveyors_accounts = ["3.2"]
-    m.fs.R_conveyors = UnitModelBlock()
+    # TODO: Add bounds to flow_mass_product by converting it to a variable
 
-    flow_3_2 = value(
-        units.convert(
-            flowsheet.fs.roaster.flow_mass_product[0]
-            + flowsheet.fs.roaster.flow_mass_dust[0],
-            to_units=units.ton / units.hr,
-        )
-    )
-    m.fs.R_conveyors.throughput = Var(initialize=flow_3_2, units=units.ton / units.hr)
-    m.fs.R_conveyors.throughput.fix()
-    m.fs.R_conveyors.costing = UnitModelCostingBlock(
-        flowsheet_costing_block=m.fs.costing,
-        costing_method=QGESSCostingData.get_REE_costing,
-        costing_method_arguments={
-            "cost_accounts": R_conveyors_accounts,
-            "scaled_param": m.fs.R_conveyors.throughput,
-            "source": 1,
-            "n_equip": 1,
-            "scale_down_parallel_equip": False,
-            "CE_index_year": CE_index_year,
-        },
-    )
+    # 3.2 is UKy Roasting - Conveyors
```

Comment:
Commented code - why is this being left out if other balance-of-plant equipment is included?

Reply:
This particular case is being left out because `m.fs.roaster.flow_mass_product` introduces a warning (potential evaluation error) in the costing, so the tests don't pass. From what I can tell, it's because this mass flow needs a lower bound, but it's set up as an Expression rather than a Var. I've played around with ways to add bounds to Expressions, but I couldn't find any methods that worked. Converting it to a Var would allow us to set bounds, but that would likely require a decent number of changes to the roaster model, which is something @jinliangma would need to handle, I imagine.

Reply:
There is no way to add bounds to an Expression. What is the specific error message you see?

Reply:
There's a DiagnosticToolbox check to ensure there are no structural issues with the `add_costing` function, but there's a potential evaluation error with this one equipment. 
![image](https://github.com/prommis/prommis/assets/96305519/64a53dfc-37f6-4398-b069-410cfc418630)

Reply:
There is a flag to get the test to skip potential evaluation errors. However, that also looks like something where there should be a bound added, so open a new issue and tag @jinliangma to fix that (and remove the flag to skip the evaluation errors when it is fixed).

Reply:
`assert_no_structural_issues(ignore_evaluation_errors=True)`